crates/daphne-server/src/storage_proxy_connection/kv/mod.rs

```diff
@@ -252,28 +252,74 @@ impl<'h> Kv<'h> {
     }
 
     #[tracing::instrument(
-        name = "kv_put",
+        name = "kv_delete",
```

Comment:
Why is this changing? I'm sure there's a good reason, but I don't understand it :)

Reply:
Why was this resolved? It should be `kv_put`

Reply:
Sorry about that, changed only one instance of this error and resolved. Corrected